src/core/tests/library/RuntimeCompositor.py

```diff
@@ -101,8 +104,6 @@ def __init__(self, argv=Constants.DEFAULT_UNSPECIFIED_VALUE, legacy_mode=False,
         self.configure_patching_processor.auto_assess_service_manager.remove_service = self.mock_remove_service
         self.backup_remove_timer = self.configure_patching_processor.auto_assess_timer_manager.remove_timer
         self.configure_patching_processor.auto_assess_timer_manager.remove_timer = self.mock_remove_timer
-        self.backup_os_getenv = os.getenv
-        os.getenv = self.getenv_telemetry_enabled
 
```

Comment:
ensure to remove restting os.getenv from self.backup_os_getenv also, if done

Reply:
Done - core tests don't reset os.getenv